step-framework-parent/step-framework-collections/src/main/java/step/core/accessors/Accessor.java

```diff
@@ -55,8 +55,10 @@ public interface Accessor<T extends AbstractIdentifiableObject> {
 	 * @return the object
 	 */
 	T findByCriteria(Map<String, String> criteria);
-	
-	/**
+

```

Comment:
Formatting indentation issue
Use directly a List, no need of collection and use "import" rather than having the package name here